services/common-api/app/controllers/v1/api_base_controller.rb

```diff
@@ -45,7 +45,8 @@ def is_platform_user?
 
     def is_admin?
       user_role = current_platform.user_roles.find_by_user_id jwt_user_id
-      user_role.role_name == 'admin'
+      role_name = user_role[:role_name].presence || ''
```

Comment:
if user_role is nil the [:role_name] will break.

use:
role_name = user_role.try(:[], :role_name)

Reply:
I replaced the code.